model/src/main/kotlin/utils/DatabaseUtils.kt

```diff
@@ -110,7 +110,7 @@ object DatabaseUtils {
     /**
      * Start a new asynchronous transaction to execute the given [statement] on this [Database].
      */
-    suspend fun <T> Database.asyncTx(statement: suspend Transaction.() -> T): Deferred<T> =
+    suspend fun <T> Database.startTransactionAsync(statement: suspend Transaction.() -> T): Deferred<T> =
```

Comment:
I would remove the `start` prefix because it doesn't add much value and only makes it longer. Renaming should also be aligned with the other `tx` function for consistency. So e.g. name them `transaction()` and `transactionAsync()`. Or maybe call them `transaction()` and `suspendedTransaction()` to align with exposed: https://github.com/JetBrains/Exposed/wiki/Transactions#working-with-coroutines